Sources/EpoxyBars/BarInstaller/BarStackView.swift

```diff
@@ -316,4 +385,76 @@ public class BarStackView: UIStackView {
     }())
   }
 
+  /// Update the background of wrappers according to the current selection.
+  private func updateHighlighting() {
+    for wrapper in wrappers {
+      wrapper.updateSelection(isSelected: selectedWrapper === wrapper)
+    }
+  }
+
+}
+
+// MARK: - StyledView
+
+extension BarStackView {
+
+  /// The style to be used.
+  public struct Style: Hashable {
+
+    // MARK: Lifecycle
+
+    public init(
+      selectedBackgroundColor: UIColor? = nil,
+      zOrder: BarStackView.ZOrder = .topToBottom)
+    {
+      self.selectedBackgroundColor = selectedBackgroundColor
+      self.zOrder = zOrder
+    }
+
+    // MARK: Internal
+
+    static var topToBottom: Self {
+      .init(zOrder: .topToBottom)
+    }
+
+    static var bottomToTop: Self {
+      .init(zOrder: .bottomToTop)
+    }
+
+    /// The selected background color to apply.
+    var selectedBackgroundColor: UIColor?
+
+    /// The order that the bars are arranged on the Z axis.
+    var zOrder: BarStackView.ZOrder
```

Comment:
Let's make these public since the memberwise init is public
```suggestion
    /// The selected background color to apply.
    public var selectedBackgroundColor: UIColor?

    /// The order that the bars are arranged on the Z axis.
    public var zOrder: BarStackView.ZOrder
```